container/container.go

```diff
@@ -568,3 +568,21 @@ func (container *Container) InitDNSHostConfig() {
 		container.HostConfig.DNSOptions = make([]string, 0)
 	}
 }
+
+// UpdateMonitor updates monitor configure for running container
+func (container *Container) UpdateMonitor(hostConfig *containertypes.HostConfig) error {
```

Comment:
Yes, it's OK.
I thought we may need other contents of hostConfig in `UpdateMonitor`, that's why I passed the whole hostConfig.
Just checked and it seems we will hardly need other part. Passing only a policy is OK.